sdk/search/azure-search-index/azure/search/index/_generated/aio/_configuration_async.py

```diff
@@ -0,0 +1,64 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Code generated by Microsoft (R) AutoRest Code Generator (autorest: 3.0.6185, generator: {generator})
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+
+from azure.core.configuration import Configuration
+from azure.core.pipeline import policies
+
+VERSION = "unknown"
+
+
+class SearchIndexClientConfiguration(Configuration):
+    """Configuration for SearchIndexClient
+    Note that all parameters used to create this instance are saved as instance
+    attributes.
+
+    :param search_service_name: The name of the search service.
+    :type search_service_name: str
+    :param search_dns_suffix: The DNS suffix of the search service. The default is search.windows.net.
+    :type search_dns_suffix: str
+    :param index_name: The name of the index.
+    :type index_name: str
+    """
+
+    def __init__(self, search_service_name, search_dns_suffix, index_name, **kwargs):
+        if search_service_name is None:
+            raise ValueError("Parameter 'search_service_name' must not be None.")
+        if search_dns_suffix is None:
+            raise ValueError("Parameter 'search_dns_suffix' must not be None.")
+        if index_name is None:
+            raise ValueError("Parameter 'index_name' must not be None.")
+        super(SearchIndexClientConfiguration, self).__init__(**kwargs)
+
+        self.search_service_name = search_service_name
+        self.search_dns_suffix = search_dns_suffix
+        self.index_name = index_name
+        self.api_version = "2019-05-06"
+        self._configure(**kwargs)
+        self.user_agent_policy.add_user_agent(
+            "azsdk-python-searchindexclient/{}".format(VERSION)
+        )
+
+    def _configure(self, **kwargs):
+        self.user_agent_policy = kwargs.get(
```

Comment:
autorest should add the RequestIdPolicy to the pipeline configuration, that fix might not be in yet.

Reply:
I'll update the generated code with new autorest version today.